app/parse.py

```diff
@@ -8,5 +13,46 @@ class Course:
     duration: str
 
 
-def get_all_courses() -> list[Course]:
-    pass
+def parse_courses(course_elements: Tag) -> Course:
+    name = course_elements.select_one(".typography_landingH3__vTjok").text
```

Comment:
The CSS class name `.typography_landingH3__vTjok` is hardcoded. If the website's structure changes, this could break the parsing logic. Consider making this more robust by using a configuration file or a more flexible selection method.